src/website/shared/logs/logs.py

```diff
@@ -186,4 +186,42 @@ def get_dict(
             else:
                 grouped_activity_log[suggestion_id] = [event]
 
-        return grouped_activity_log
+        # Second pass to fold repeated package actions by user
```

Comment:
```suggestion
        # Second pass to fold repeated package actions by user,
        # needed because with htmx we're sending item-wise changes that we still want to display in bulk
```